codegen/model/resource.go

```diff
@@ -56,6 +56,11 @@ type Group struct {
 	// Should we generate kubernetes manifests?
 	RenderManifests bool
 
+	// Should we generate validating schemas for CRDs?
+	// NOTE: proto int64's cannot currently be used with generated validation schemas.
+	// See https://github.com/solo-io/skv2/issues/146 for details.
+	RenderValidationSchemas bool
```

Comment:
it would be nice to add a test for this (in cmd_test.go), and would also be nice to enable it for the skv2 multicluster api group